blocks/header/header.css

```diff
@@ -466,76 +479,60 @@
   }
 
   .header .header__menu-content .header__menu-heading {
+    padding: 0;
+    margin-bottom: 24px;
+    border-bottom: none;
+    font-family: var(--font-family-heading);
     font-size: 24px;
     letter-spacing: var(--f-heading-5-letter-spacing);
     line-height: var(--f-heading-5-line-height);
-  }
-
-  .header__category-subtitle {
-    margin-bottom: 16px;
-  }
-
-  .header__category-subtitle:not(:empty) {
-    font-size: 12px;
-    letter-spacing: var(--f-caption-letter-spacing);
-    line-height: var(--f-caption-line-height);
-    margin-bottom: 40px;
-    color: var(--c-grey-600);
+    color: var(--c-main-black);
   }
 
   .header .header__category-items {
     margin: 0;
-    column-gap: 12px;
-    row-gap: 16px;
+    column-gap: 16px;
+    row-gap: 24px;
     display: flex;
     flex-wrap: wrap;
   }
 
   /* stylelint-disable-next-line no-descending-specificity */
-  .header__category-item .header__link {
-    display: inline;
-    padding: 0;
-    border-bottom: solid 1px transparent;
-    text-wrap: balance;
+  .header__category-item picture + .header__link {
+    margin-top: 8px;
   }
 
-  .header__category-item a:first-of-type {
-    display: inline;
+  /* Expand link to be placed over the image above */
+  .header__category-item picture + a::before {
+    content: '';
+    position: absolute;
+    inset: 0 0 auto;
+    aspect-ratio: 5/4;
   }
 
   .header__category-item a::after {
     content: '';
     background-image: url('/icons/chevron-down.svg');
```

Comment:
Suggestion - we can use a bg shorthand here 
background: url('/icons/chevron-down.svg') no-repeat center/cover;